src/test/resources/features/delete_account.feature

```diff
@@ -0,0 +1,17 @@
+Feature: Accounts
+
+  Background:
+    Given I sign in as "owner"
+    And I click on Accounts
+    And I click new account button
+    And I create "RPFH account" account
+    And I validate the account name is "RPFH account"
+
+
+  Scenario: Delete Account
+    When I click settings tab
+    And I click delete link
+    Then I validate the "RPFH account" is not displayed
+    And I validate a delete "RPFH account" message is displayed
+    And I validate "RPFH account" is not displayed on create project form
+    And I validate "RPFH account" is not displayed on "TestProject" more tab
```

Comment:
your test should run in different environments
so now this is failing because it doesn't exist that project "TestProject"